### DISCUSSION TITLE
Is it possible to get data from Redmine and receive data import to NocoDb by using API

### DISCUSSION THREAD

**Discussion by anothaiyoyo** [nocodb/nocodb#4085] [Q&A]
Is it possible to get data from Redmine (like Issues and projects ) , and import data to NocoDb by using API , any suggestion or how to do it . 

Thank you .

**Comment by o1lab** [comment]
Not at the moment, could you please expand on the usecase.

**Reply by anothaiyoyo** [reply]
My topic that have got is , when The Redmine update issues or projects it can automatically get request data from Redmine and update to NocoDB in real time , is it possible to do that ?

**Comment by wingkwong** [comment]
Why using API? In NocoDB, you can connect the Redmine database directly. Or you can export the data from Redmine in spreadsheet / csv format and import it to NocoDB.

**Reply by anothaiyoyo** [reply]
Thank you for your advice ,  I already did that it's worked nicely , but My topic that have got is ,  when The Redmine update issues or projects it can automatically get request data from Redmine and update to NocoDB in real time  , is it possible to do that  ?

**Reply by o1lab** [reply]
As long as redmine database is updated - it will be reflected within nocodb

**Reply by anothaiyoyo** [reply]
Thank you very much !
